docker/manage

```diff
@@ -267,6 +267,30 @@ pylint() {
   $PYLINT greenlight-agent/src/*.py
 }
 
+initDockerBuildArgs() {
+  dockerBuildArgs=""
+
+  # HTTP proxy, prefer lower case
+  if [[ "${http_proxy}" ]]; then
+    dockerBuildArgs=" ${dockerBuildArgs} --build-arg http_proxy=${http_proxy}"
+  else
+    if [[ "${HTTP_PROXY}" ]]; then
+      dockerBuildArgs=" ${dockerBuildArgs} --build-arg http_proxy=${HTTP_PROXY}"
+    fi
+  fi
+
+  # HTTPS proxy, prefer lower case
+  if [[ "${https_proxy}" ]]; then
+    dockerBuildArgs=" ${dockerBuildArgs} --build-arg https_proxy=${https_proxy}"
+  else
+    if [[ "${HTTPS_PROXY}" ]]; then
+      dockerBuildArgs=" ${dockerBuildArgs} --build-arg https_proxy=${HTTPS_PROXY}"
+    fi
+  fi
+
+  echo ${dockerBuildArgs}
+}
```

Comment:
Since this is just for the http proxy argument, I am wondering if it would be best to:
- rename the function to be more specific
- make it more generic, so any docker build argument could be passed (e.g.: setting an environment variable with the list of arguments to apply).

I'll let @WadeBarnes chime in since he may have done something like that before.

Reply:
The name is fine, it's for `--build-arg`s